src/go/rpk/pkg/cli/cmd/status.go

```diff
@@ -259,30 +274,45 @@ func getKafkaInfo(conf config.Config, out chan<- [][]string) error {
 	client, err := kafka.InitClientWithConf(&conf, addr)
 	if err != nil {
 		out <- [][]string{}
+		kafkaInfoCh <- kInfo
 		return errors.Wrap(err, "Error initializing redpanda client")
 	}
 	admin, err := sarama.NewClusterAdminFromClient(client)
 	if err != nil {
 		out <- [][]string{}
+		kafkaInfoCh <- kInfo
 		return errors.Wrap(err, "Error initializing redpanda client")
 	}
 	defer admin.Close()
 	topics, err := topicsDetail(admin)
 	if err != nil {
 		out <- [][]string{}
+		kafkaInfoCh <- kInfo
 		return errors.Wrap(err, "Error fetching the Redpanda topic details")
 	}
-	if len(topics) == 0 {
+	if send {
```

Comment:
I think we can *always* display/gather the kafkaInfo, doesn't have to be gated IMO.

Reply:
I did this to be consistent with https://github.com/vectorizedio/redpanda/pull/329

Reply:
makes snese.